code/modules/mob/living/carbon/human/descriptors/_descriptors.dm

```diff
@@ -56,6 +56,8 @@
 	if(ishuman(me) && !skip_species_mention)
 		var/mob/living/carbon/human/H = me
 		var/use_name = "\improper [H.species.name]"
+		if(H.custom_species)
```

Comment:
Leave a comment next to this so I know what this change is from next time we merge w/ release updates.

Reply:
Just a //Mithra Additions will work.

Reply:
Alternatively, one could consider making a copy of the file in the modular folder, and setting the .DME to call *that* version instead of the mainline.

Reply:
I would defintley have done the second thing if I was confident that I knew how to without breaking everything (I'm not)